[fix][error] C++ style comments are not allowed in ISO C90

wuhanstudio opened the pull request:
Compiling in Ubuntu 20.04 with gcc 9.3.0 throws an error:

    ../../nuklear.h:16579:17: error: C++ style comments are not allowed in ISO C90
    16579 |                 // Need to zero this, or it will carry over from a previous
          |                 ^

What I did:

```
git clone https://github.com/Immediate-Mode-UI/Nuklear
cd demo/x11_rawfb/
make
```

Hejsil commented:
You'll need to apply this change to one of the files in `src` and then [generate the `nuklear.h`](https://github.com/Immediate-Mode-UI/Nuklear/tree/master/src#file-packer). You also need to update the  
[changelog](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG) and [package.json](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/package.json).

Otherwise this looks good and will be merged once this is done :+1:

wuhanstudio commented:
> You'll need to apply this change to one of the files in `src` and then [generate the `nuklear.h`](https://github.com/Immediate-Mode-UI/Nuklear/tree/master/src#file-packer). You also need to update the
> [changelog](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG) and [package.json](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/package.json).
> 
> Otherwise this looks good and will be merged once this is done 👍

Thanks for your suggestion. I did

> You'll need to apply this change to one of the files in `src` and then [generate the `nuklear.h`](https://github.com/Immediate-Mode-UI/Nuklear/tree/master/src#file-packer). You also need to update the
> [changelog](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG) and [package.json](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/package.json).
> 
> Otherwise this looks good and will be merged once this is done 👍

Thanks for reminding me. Just updated changelog and package.json, and regenerated nuklear.h.

Since it's just a minor error, I changed from 4.06.0 to 4.06.1 in changelog.

Thanks again for your suggestion.
